src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -492,6 +506,10 @@ public static void main(final String[] args) {
             messages.add("The number of tags given in TAG_PER_MOLECULAR_INDEX does not match the number of molecular indexes in READ_STRUCTURE");
         }
 
+        if ((FIVE_PRIME_ADAPTER != null && THREE_PRIME_ADAPTER == null) || (THREE_PRIME_ADAPTER != null && FIVE_PRIME_ADAPTER == null)) {
+            messages.add("Either both or neither of THREE_PRIME_ADAPTER and FIVE_PRIME_ADAPTER must be set.");
```

Comment:
Lifted verbatim over from MarkIlluminaAdapters. I'll change both, but I am not interested in getting too nitpicky about it.